packages/contract-schema/test/schema.js

```diff
@@ -3,20 +3,43 @@ var assert = require("assert");
 
 var MetaCoin = require("./MetaCoin.json");
 
-describe("Schema", function() {
-  it("validates correct input", function() {
+describe("Schema", function () {
+  it("validates correct input", function () {
     Schema.validate(MetaCoin);
   });
 
-  it("throws exception on invalid input", function() {
+  it("throws exception on invalid input", function () {
     var invalid = {
       abi: -1
     };
 
     try {
       Schema.validate(invalid);
     } catch (err) {
-      var abiErrors = err.errors.filter(function(error) {
+      var abiErrors = err.errors.filter(function (error) {
+        return error.dataPath === ".abi";
+      });
+      assert(abiErrors);
+    }
+  });
+
+  it("validates a correct input as part of normalization", function () {
+    Schema.normalize(MetaCoin, {
+      validate: true
+    });
```

Comment:
I think this is fine as it is; if it throws it throws.  Applying this suggestion would just make the stacktraces less useful.

Reply:
@haltman-at can you please explain how would the stacktrace be less useful here? I don't follow :/

Reply:
If you let the call just fail, then you get a stacktrace from within the call, where the error occurs.  If you catch the error and then assert false, you get a stacktrace that originates at the assertion.

Reply:
I agree with @haltman-at on this one. Providing we want the valuable stacktrace, even if we would wrap the expression in a try/catch block and use assert like this: 
```js
try {
  Schema.normalize(MetaCoin, {
    validate: true
  });
} catch(err) { 
  assert(false, err);
}
```
It would have the same effect as an uncaught exception, it would still render the stacktrace.